pyo3-bytes/src/bytes.rs

```diff
@@ -341,7 +404,7 @@ impl<'py> FromPyObject<'py> for PyBytes {
 ///
 /// This also implements AsRef<[u8]> because that is required for Bytes::from_owner
 #[derive(Debug)]
-struct PyBytesWrapper(Option<PyBuffer<u8>>);
+pub struct PyBytesWrapper(Option<PyBuffer<u8>>);
```

Comment:
Why should this be pub?

Reply:
whoops. This is a remnant of me trying to figure out how to make taking a `pyo3_bytes::PyBytes` or `pyo3::...::PyBytes` object when you only need `&[u8]` -- I was going to write up my findings and send them to you -- basically taking a `PyBytes` when you only need a ref to the slice has a large perf hit.

Reply:
> basically taking a PyBytes when you only need a ref to the slice has a large perf hit

hmm, can you expand on this?

Reply:
BABOOM
https://github.com/developmentseed/obstore/issues/251